roles/mattermost/tasks/main.yml

```diff
@@ -0,0 +1,30 @@
+- name: Deploy mattermost + matterbridge
+  block:
+      - file:
+          path: "{{ docker_data_folder }}/mattermost"
+          state: directory
```

Comment:
What owner/group/mode is used if it's unspecified?

Reply:
should be always root, but make it explicit is usually a good idea 
34f4bfb